lighthouse-treemap/app/src/main.js

```diff
@@ -641,10 +666,42 @@ function injectOptions(options) {
   document.head.append(scriptEl);
 }
 
+/**
+ * @param {import('../../../lighthouse-core/lib/i18n/locales').LhlMessages} localeMessages
+ */
+function getStrings(localeMessages) {
+  const strings = /** @type {Record<string, string>} */ ({});
+  for (const icuMessageId of Object.keys(localeMessages)) {
+    const [filename, varName] = icuMessageId.split(' | ');
+    if (!filename.endsWith('util.js')) throw new Error(`Unexpected message: ${icuMessageId}`);
+
+    const key = /** @type {keyof LH.I18NRendererStrings} */ (varName);
```

Comment:
definitely not `LH.I18NRendererStrings` :)

Can chopping off the path be moved to the build step? If we don't use them anyways, it would certainly cut down that 40KiB to something much smaller :)

Reply:
good idea